drivers/avr/i2c_master.c

```diff
@@ -7,8 +7,11 @@
 
 #include "i2c_master.h"
 #include "timer.h"
+#include "config.h"
```

Comment:
Is the config.h necessary here? 

I think that they get included by the make script.